src/components/Layout.tsx

```diff
@@ -2,7 +2,10 @@ import Header from "./Header";
 import SideMenu from "./SideMenu";
 import styled from "styled-components";
 import OverviewPage from "../pages/Overview/OverviewPage";
-import EditOrganization from "../pages/Organizations/EditOrganization";
+import {
+  EditOrganization,
+  AddOrganization,
+} from "../pages/Organizations/EditOrganization";
```

Comment:
You might not like this organization @brunellaa . I separated them out because it was easier for me to hack in what we needed here. Happy to restructure post-release!